test/WebSites/BasicWebSite/StartupWithSessionTempDataProvider.cs

```diff
@@ -6,13 +6,14 @@
 
 namespace BasicWebSite
 {
-    public class StartupWithCookieTempDataProvider
+    public class StartupWithSessionTempDataProvider
```

Comment:
Rename this file and class to `Startup` and the other to `StartupWithSessionTempDataProvider` as currently this class does not have the session services or middleware